examples/Rakefile

```diff
@@ -1,41 +1,111 @@
 require 'fastly_nsq'
 require 'fastly_nsq/rake_task'
 
-##
-# Both topic and channel will need to be passed into the rake task when
-# it is called.
+#-------------------------------------------------------------------------------
+# You are required to to set the topics in MessageProcessor.topics as an array:
 #
-# ex: `rake listen_task[my_topic,my_channel]`
-# topic will be 'my_topic' and channel will be 'my_channel'
+# class MessageProcessor
+#   def self.topics
+#     ['customer_created', 'customer_deleted']
+#   end
 #
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel])
-
-##
-# In Rails, you can include the environment like so:
-#
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel] => :environment)
-
-##
-# Topic and channel can also be preset in the task by passing a block and
-# assigning value like so. This is useful if you want a single purpose
-# listener task.
+#   # ...
+# end
 #
-# ex: `rake listen_task`
-# topic will be 'some_topic' and channel will be 'some_channel'
+# A channel will need to be passed in
+# to the Rake task
+# when it is called.
```

Comment:
### Short answer
"Semantic linefeeds", which means "one line per thought/clause". The goal is to reduce diffs, much like the trailing comma in a multi-line Ruby list. Also it separates ideas (for me) and is easier to write prose properly (again, for me). Markdown doesn't care and will jam them all into one paragraph.

### Long answer

The above, plus this is something which goes back to the Bell Labs UNIX philosophy of dealing only in plain text:

> Hints for Preparing Documents

> Most documents go through several versions (always more than you expected) before they are finally finished. Accordingly, you should do whatever possible to make the job of changing them easy.

> First, when you do the purely mechanical operations of typing, type so subsequent editing will be easy. Start each sentence on a new line. Make lines short, and break lines at natural places, such as after commas and semicolons, rather than randomly. Since most people change documents by rewriting phrases and adding, deleting and rearranging sentences, these precautions simplify any editing you have to do later.

> — Brian W. Kernighan, Bell Labs, 1974
> From "UNIX for Beginners" (http://miffy.tom-yam.or.jp/2238/ref/beg.pdf)